src/base/preferences.cpp

```diff
@@ -449,6 +451,36 @@ void Preferences::setWebUiLocalAuthEnabled(bool enabled)
     setValue("Preferences/WebUI/LocalHostAuth", enabled);
 }
 
+bool Preferences::isWebUiAuthIPBypassEnabled() const
+{
+    return value("Preferences/WebUI/EnableAuthIPBypass", false).toBool();
+}
+
+void Preferences::setWebUiAuthIPBypassEnabled(bool enabled)
+{
+    setValue("Preferences/WebUI/EnableAuthIPBypass", enabled);
+}
+
+QList<QPair<QHostAddress, int> > Preferences::getWebUiAuthBypassSubnets() const
```

Comment:
>There's no coding guideline on it.

We can't set all the rules verbatim! There are some things in common... for Example, we don't usually put spaces inside the parentheses (in this case, the brackets have similar semantics). At least the brackets here are pairs of things, and they should look flipped relative to their content (i.e. no whitespace after the opening bracket then must not be before the closing).

But if some things not so obvious, we can add some new rules in Coding Guidelines.
>This is done for readability

IMO, this is against the readability.